test/functional/inabox/test-utils.js

```diff
@@ -52,10 +68,21 @@ describes.fakeWin('inabox-utils', {}, env => {
     setTimeout(() => timeDeferred.resolve(), 10);
     return timeDeferred.promise.then(() => {
       expect(dispatchEventStub).to.be.calledOnce;
-      expect(initCustomEventStub.withArgs('amp-ini-load', true, false, null))
+      expect(initCustomEventStub.withArgs('amp-ini-load'))
           .to.be.calledOnce;
       expect(parentPostMessageStub.withArgs('amp-ini-load', '*'))
           .to.be.calledOnce;
     });
   });
+
+  it('Should not return an a4aId if no a4a meta tag in head', () => {
+    expect(getA4AId(env.win)).to.be.not.ok;
+  });
+
+
+  it('Should be able to get the a4aId if on the document', () => {
+    addA4AMetaTagToDocument();
+    expect(getA4AId(env.win)).to.be.ok;
+    removeA4AMetaTagToDocument();
```

Comment:
you don't need to clean up, given every iframe that runs the test is new.

Reply:
Done.